docs/content/mongocxx-v3/installation/windows.md

```diff
@@ -33,6 +33,9 @@ or newer. The MongoDB C Driver supports TLS 1.1 on Linux if OpenSSL is
 at least version 1.0.1. On macOS and Windows, the C Driver uses native
 TLS implementations that support TLS 1.1.
 
+If you are compiling on Windows, you should also use the
```

Comment:
The word "use" may cause users to wonder if `ENABLE_EXTRA_ALIGNMENT` should be enabled or disabled. You could use the word "disable" or specify the value `0` like the variables below or both.

Reply:
Sounds good, used the word "disable" and clarified setting `DENABLE_EXTRA_ALIGNMENT=0`.